common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -74,6 +64,11 @@ private static void initializeFeature(Feature feature) {
             return;
         }
 
+        // OverlayManager event listener setup
+        if (featureClass.isAnnotationPresent(OverlayFeature.class)) {
```

Comment:
I think I understand what P0ke means now. Or maybe that's not what he means, but that's what I suddenly see as a problem. :-)

We don't need the `@OverlayFeature` annotation, as far as I can see. If a feature has Overlay fields with the `@OverlayInfo` annotation, that is enough. We don't have to tag the class as well. That's just confusing, and adds no value, we only risk that we miss that we miss it and the Overlays do not work. If a class has Overlays with OverlayInfo, then it is an "overlay feature", otherwise not.

(The thing that got me confused is that the P0ke was talking about registering in the init function, but actually that registering is based on annotations as well, just that it is field annotations instead of class annotations. So I misunderstood this comment as to mean we should go back to some kind of call "initOverlays()" in the Feature, which I then argued against :-D)